src/toxcore/dht/packet.rs.old

```diff
@@ -1,1629 +0,0 @@
-/*
```

Comment:
This commit is named as `refactor(dht): remove quickcheck`. Don't remove packet.rs.old in it